app/controllers/provider/admin/backend_apis/metrics_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApis::MetricsController < Provider::Admin::BackendApis::BaseController
+  before_action :find_metric, except: %i[new create index]
+  before_action :build_metric, only: %i[new create]
+
+  activate_menu :backend_api, :methods_metrics
+  sublayout 'api/service'
+
+  def create
+    # TODO
```

Comment:
Good for me as long as we don't forget 👍 😄

Reply:
We tend to forget about our TODOs, maybe reinforce our memory with a jira issue?

Reply:
There is a Jira for this already: https://issues.jboss.org/browse/THREESCALE-3089. No way to forget as it is mandatory for feature complete.